bindings/wasm/.eslintrc.js

```diff
@@ -15,6 +15,7 @@ module.exports = {
         sourceType: 'module',
     },
     rules: {
+        '@typescript-eslint/no-var-requires': 0,
```

Comment:
examples/node.js gives errors on imports, but changing to requested makes example break